.github/workflows/core-dependency-update.yml

```
@@ -59,4 +59,28 @@
+          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}"
+  
+  codecov:
+    needs: update-dependency
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)